### ISSUE TITLE
Landis&gyr 

### ISSUE THREAD

**Issue by perfredrikskold** [home-assistant/home-assistant.github.io#29132]
### Feedback

Om wondering if the meter is connected to 230v do it still take 30 minutes of battery time? If not or if it does I think this info should be presenter under the integration page!

### URL

https://www.home-assistant.io/integrations/landisgyr_heat_meter/

### Version

2023.9.3

**Comment by vpathuis** [comment]
True, the battery will only go down if it actually uses a battery. The integration currently is not tested for units on power supply. I'm sure it'll work though, and I don't have any problems with a change reflecting that in the docs. 

Note: I've tried a change for supporting units on power supply a while back, but that was rejected: https://github.com/home-assistant/core/pull/90798
